### PR TITLE
eegrasp.py code refactoring

### PR DESCRIPTION
**Pull request by lcortesg**

### CODE REVIEW

File: eegrasp/utils.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Parameters
+    ----------
+    coordinates : ndarray | None
+        N-dim array with position of the electrodes. If `None` the class
+        instance will use the coordinates passed at initialization. Default
+        is `None`.
+    method : string
+        Options are: 'Euclidean'. Method used to compute the distance matrix.
+    normalize : bool
+        If True, the distance matrix will be normalized before being
```
[the remark this answers is not in the document]
Comment (lcortesg) [reply eegrasp/utils.py:46]: I didn't write the functions, just moved the around for easier parsing, will discuss with @julioRodino about a better description.

File: eegrasp/utils.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    from scipy.spatial import distance_matrix
+
+    distance = np.zeros([pos.shape[0], pos.shape[0]],
+                        dtype=np.float64)  # Alocate variable
+    pos = pos.astype(float)
+    distance = distance_matrix(pos, pos)
+    return distance
+
+
+def compute_distance(coordinates=None, method='Euclidean', normalize=True, coord0=None):
```
[the remark this answers is not in the document]
Comment (lcortesg) [reply eegrasp/utils.py:34]: Due to some class methods changing in-class attributes, new default variables had to be added, will modify accordingly.

File: eegrasp/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (aweinstein) [reply eegrasp/utils.py:46]: OK. I'll create an issue to address this specifically.

File: eegrasp/utils.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        assigned to the class' instance without normalization.
+
+    Returns
+    -------
+    distances : ndarray
+        Distances to be used for the graph computation.
+    """
+
+    # If passed, used the coordinates argument
+    if isinstance(coordinates, type(None)):
```
[the remark this answers is not in the document]
Comment (aweinstein) [reply eegrasp/utils.py:57]: OK. I'll create an issue to address this specifically.

File: eegrasp/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (lcortesg) [reply eegrasp/utils.py]: Solved by doing the parameter verification inside the class methods, and passing them to the outside functions as valid variables.

**Review by aweinstein — APPROVED** [review]

**Pull request merged**